src/app/views/communities/view/overview/overview.vue

```diff
@@ -113,6 +30,11 @@
 					/>
 					<app-communities-view-overview-nav-edit v-else class="-nav" :community="community" />
 				</app-scroll-affix>
+
+				<app-button v-if="Screen.isXs" class="-mobile-info-btn" @click="onClickAbout" block>
```

Comment:
use Screen.isMobile to be consistent with the conditions where the sidebar shows.